src/test/java/uk/gov/hmcts/divorce/systemupdate/schedule/conditionalorder/SystemNotifyJointApplicantCanSwitchToSoleTaskTest.java

```diff
@@ -100,8 +95,6 @@ void setUp() {
     @Test
     void shouldSubmitNotifySwitchToSoleEventWhenJointConditionalOrderOverdueFromApplicant2() {
 
-        setMockClock(clock, NOW);
-
         Map<String, Object> data = new HashMap<>();
         data.put("coApplicant1IsSubmitted", "Yes");
         data.put("coApplicant1SubmittedDate", "2022-07-07T00:00:00.000Z");
```

Comment:
I think these values are not used anywhere, so I guess these can be removed .. just to avoid any confusion in submitted date, as we are using now() value in the mapper mock response object.